shell/platform/android/android_context_gl.cc

```diff
@@ -196,13 +155,39 @@ AndroidContextGL::~AndroidContextGL() {
     LogLastEGLError();
   }
 
-  if (!TeardownSurface(environment_->Display(), surface_)) {
-    FML_LOG(ERROR)
-        << "Could not tear down the EGL surface. Possible resource leak.";
+  if (!TeardownContext(environment_->Display(), resource_context_)) {
+    FML_LOG(ERROR) << "Could not tear down the EGL resource context. Possible "
+                      "resource leak.";
     LogLastEGLError();
   }
 }
 
+// For onscreen rendering.
+EGLSurface AndroidContextGL::CreateOnscreenSurface(
+    fml::RefPtr<AndroidNativeWindow> window) {
+  // The configurations are only required when dealing with extensions or VG.
```

Comment:
What configurations? This comment  is now out of date since you moved the config_ initialization maybe?